mirrord/intproxy/src/proxies/incoming.rs

```diff
@@ -270,6 +278,53 @@ impl IncomingProxy {
                     interceptor.send(req).await;
                 }
             }
+            DaemonTcp::HttpRequestChunked(req) => {
+                tracing::error!("{:?}", req);
+                match req {
+                    ChunkedRequest::Start(req) => {
+                        let (tx, rx) = mpsc::channel::<InternalHttpBodyFrame>(12);
+                        let http_stream = StreamingBody::new(rx);
+                        let http_req = HttpRequest {
+                            internal_request: InternalHttpRequest {
+                                method: req.internal_request.method,
+                                uri: req.internal_request.uri,
+                                headers: req.internal_request.headers,
+                                version: req.internal_request.version,
+                                body: http_stream,
+                            },
+                            connection_id: req.connection_id,
+                            request_id: req.request_id,
+                            port: req.port,
+                        };
+
+                        for frame in req.internal_request.body {
+                            let _ = tx.send(frame).await;
+                        }
```

Comment:
Sending these should happen after passing the request to the interceptor. No one is receiving from the other end of the channel yet, so if you we have more than 12 frames ready, it's a deadlock.

Also, we should not ignore the send errors here. If sending fails, we should drop the sender and return early